tarantool_test.go

```diff
@@ -1986,21 +1999,298 @@ func TestComplexStructs(t *testing.T) {
 	}
 }
 
+func TestStream_Commit(t *testing.T) {
+	var req Request
+	var resp *Response
+	var err error
+	var conn *Connection
+
+	// Tarantool supports streams and interactive transactions since version 2.10.0
+	isLess, err := test_helpers.IsTarantoolVersionLess(2, 10, 0)
+	if err != nil {
+		t.Fatal("Could not check the Tarantool version")
+	}
+	if isLess {
+		t.Skip()
+	}
+
+	conn, err = Connect(server, opts)
+	if err != nil {
+		t.Fatalf("Failed to connect: %s", err.Error())
+	}
+	if conn == nil {
+		t.Fatalf("conn is nil after Connect")
+	}
+	defer conn.Close()
+
+	stream := conn.NewStream()
+
+	// Begin transaction
+	resp, err = stream.Begin()
+	if err != nil {
+		t.Fatalf("Failed to Begin: %s", err.Error())
+	}
+	if resp.Code != 0 {
```

Comment:
So a user should be checking resp.Code to verify that everything is alright?

Reply:
Well, firstly user should check error, but it would be useful to check the response code too.

Reply:
Is it possible to have no error and non-zero response code?

Reply:
I see that it is no more relevant.